daemon/inspect_fs_unix_fstab.ml

```diff
@@ -523,7 +523,7 @@ and resolve_fstab_device spec md_map os_type =
     default
   )
 
-and resolve_dev_mapper spec default =
+and resolve_lvm spec default =
```

Comment:
This renames this function, but I believe this function (renamed to `resolve_lvm`) is only called from the (added) `resolve_dev_mapper` function below.  If so, this just adds an indirection for no reason so you might as well just inline this function where it appears in the later code.

Reply:
ok will do